src/platform_impl/linux/event_loop.rs

```diff
@@ -903,7 +922,7 @@ impl<T: 'static> EventLoop<T> {
           }
           _ => match draws.try_recv() {
             Ok(id) => callback(
-              Event::RedrawRequested(RootWindowId(id)),
+              Event::RedrawRequested(RootWindowId(id), Some(textures.try_recv().unwrap())),
```

Comment:
Is it possible to send two events in an iteration? Or is it okay to just try to pull from textures channel, and send even if something is available?